packages/create-astro/src/actions/dependencies.ts

```diff
@@ -25,12 +25,26 @@ export async function dependencies(
 		await spinner({
 			start: `Dependencies installing with ${ctx.pkgManager}...`,
 			end: 'Dependencies installed',
-			while: () =>
-				install({ pkgManager: ctx.pkgManager, cwd: ctx.cwd }).catch((e) => {
-					// eslint-disable-next-line no-console
+			while: () => {
+				return Promise.reject('Unknown error').catch((e) => {
+					error('error', e);
+					error(
+						'error',
+						`Dependencies failed to install, please run ${color.bold(
+							ctx.pkgManager + ' install'
+						)} to install them manually after setup.`
+					);
+				});
+				return install({ pkgManager: ctx.pkgManager, cwd: ctx.cwd }).catch((e) => {
 					error('error', e);
-					process.exit(1);
```

Comment:
Seems like removing this + the error message should've been enough to prevent exiting

Reply:
Yes, but it's definitely **not** a good DX printing the cause of the error without giving an explanation to the user :)